09 Redux/src/reducers/membersReducer.ts

```diff
@@ -0,0 +1,18 @@
+import MemberEntity from "../api/memberEntity";
+import MemberAPI from "../api/memberAPI";
+
+// Just to show how combine reducers work, we have
+// divided into two reducers member load + member load/update/delete
+export default (state : Array<MemberEntity> = [], action) => {
+  switch (action.type) {
+    case 'MEMBERS_LOAD':
+      let members : Array<MemberEntity>;
+      members = MemberAPI.getAllMembers();
+
+      return members;
```

Comment:
On the other hand we will recieve from the real API new elements, for the fake one not, that's true.